aws_lambda_powertools/utilities/feature_flags/time_conditions.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime, timezone
+from typing import Dict, List
+
+from .schema import TimeKeys, TimeValues
+
+HOUR_MIN_SEPARATOR = ":"
+DAY_MAPPING = {
+    1: TimeValues.MONDAY,
+    2: TimeValues.TUESDAY,
+    3: TimeValues.WEDNESDAY,
+    4: TimeValues.THURSDAY,
+    5: TimeValues.FRIDAY,
+    6: TimeValues.SATURDAY,
+    7: TimeValues.SUNDAY,
+}
+
+
+def time_range_compare(action: str, values: Dict) -> bool:
+    if action == TimeKeys.CURRENT_TIME_UTC.value:
+        return _time_range_compare_current_time_utc(values)
+    elif action == TimeKeys.CURRENT_HOUR_UTC.value:
+        return _time_range_compare_current_time_utc(values)
+    # we assume it passed validation right? so no need to raise an error
+    return False
+
+
+def time_selected_days_compare(action: str, values: List[str]) -> bool:
+    if action == TimeKeys.CURRENT_DAY_UTC.value:
+        return _time_selected_days_current_days_compare(values)
+    # we assume it passed validation right? so no need to raise an error
+    return False
+
+
+def _time_selected_days_current_days_compare(values: List[str]) -> bool:
+    current_day_number: datetime = datetime.now(timezone.utc).isoweekday()
+    return DAY_MAPPING.get(current_day_number, "") in values
```

Comment:
```suggestion
    current_day: datetime = datetime.now(timezone.utc).strftime('%A').lower()
    return current_day in values
```

Maybe this to get the name of the week directly?